shared/actions/json/signup.json

```diff
@@ -1,58 +1,64 @@
 {
   "prelude": ["import HiddenString from '../util/hidden-string'"],
   "actions": {
-    "checkInviteCode": {
```

Comment:
this and actions/signup.js are basically a big port, mostly teh same but maybe hard to see in the inline diff view